src/models/accounts.rs

```diff
@@ -12,17 +12,12 @@ pub struct Account {
 }
 
 impl Account {
-    pub fn new(
-        account_id: String,
-        index: i32,
-        receipt_id: String,
-        timestamp: BigDecimal
-    ) -> Self {
+    pub fn new(account_id: String, index: i32, receipt_id: String, timestamp: BigDecimal) -> Self {
```

Comment:
This whole `new` implementation does not look useful. You may just as well initialize the `Account` struct in place, and I would prefer to do that over `new` method with positional arguments.